externs/olx.js

```diff
@@ -4499,6 +4499,65 @@ olx.source.ImageVectorOptions.prototype.style;
 
 
 /**
+ * @typedef {{sources: Array.<ol.source.Source>,
+ *     operations: (Array.<ol.raster.Operation>|undefined),
```

Comment:
Operations can be set post-construction (with `setOperations()`), so they don't need to be provided here.